src/ert/config/gen_kw_config.py

```diff
@@ -263,9 +262,9 @@ def write_to_runpath(
             if target_file.startswith("/"):
                 target_file = target_file[1:]
             (run_path / target_file).parent.mkdir(exist_ok=True, parents=True)
-            template_file_path = (
-                ensemble.experiment.mount_point / Path(self.template_file).name
-            )
+
+            template_file_path = self.template_file
```

Comment:
Because if we want to load the one from the experiment we also need to input the path to the experiment? There was a goal at one point for the storage to be portable, but I am not sure that is the point

Reply:
Yes, we would also need the experiment (or ensemble) path as an input to `write_to_runpath()` I guess we can live with that as long as its optional

Reply:
What if we add the path `ensemble.experiment.mount_point` to `attrs` of the `xr.Dataset` we pass in?
This way we don't have to change behaviour.